WhatsNew.txt

```diff
@@ -28,4 +30,10 @@ General:
 * Removed unused 'depth' parameter from SDL_CreateRGBSurfaceWithFormat and SDL_CreateRGBSurfaceWithFormatFrom
 * Removed unused 'flags' parameter from SDL_CreateRGBSurface and SDL_CreateRGBSurfaceWithFormat
 * Removed unused 'flags' parameter from SDL_ConvertSurface and SDL_ConvertSurfaceFormat
+* SDL_CreateRGBSurfaceWithFormat() is renamed to SDL_CreateRGBSurface()
+* SDL_CreateRGBSurfaceWithFormatFrom() is renamed to SDL_CreateRGBSurfaceFrom()

```

Comment:
Can you combine the notes above into a single change and recommended application change?